apache-maven/src/assembly/maven/bin/.gitattributes

```diff
@@ -1,3 +0,0 @@
-mvn                eol=lf crlf=input
```

Comment:
@michael-o now there is no `apache-maven/src/assembly/maven/bin/mvn` file, so the original `gitattributes` is out of date.

Reply:
Didn't notice this, so I withdraw my comments.